packages/core/src/ruleset/validation/assertions.ts

```diff
@@ -0,0 +1,33 @@
+import { isPlainObject } from '@stoplight/json';
```

Comment:
copied over from `packages/core/src/ruleset/validation.ts`, the only change is the addition of `format` in `assertValidRuleset`